src/LeftNav/LeftNav.jsx

```diff
@@ -52,24 +74,47 @@ export class LeftNav extends React.Component {
     });
 
     // Find the open NavGroup so that we can render its children NavLinks in the drawer
-    const openChild = _.find(React.Children.toArray(children), child => child.props._open);
-
-    const collapsed = this.props.collapsed ? cssClass.COLLAPSED : null;
+    const openChild = _.find(this._getNonEmptyChildren(navItems), item => item.props._open);
 
     return (
-      <nav className={cssClass.CONTAINER}>
-        <div className={classnames(cssClass.TOPNAV, collapsed)}>{children}</div>
-        <ReactCSSTransitionGroup
-          transitionEnterTimeout={200}
-          transitionLeaveTimeout={300}
-          component="div"
-          transitionName="LeftNav--drawer-anim"
+      <RootCloseWrapper onRootClose={() => this._onRootClose()}>
+        <nav
+          className={classnames(
+            cssClass.CONTAINER,
+            _collapsed && cssClass.COLLAPSED,
+            openChild && cssClass.WITH_SUBNAV_OPEN,
+            className)}
         >
-          {openChild && <div className={cssClass.SUBNAV}>{openChild.props.children}</div>}
-        </ReactCSSTransitionGroup>
-      </nav>
+          <div className={classnames(cssClass.TOPNAV, _collapsed && cssClass.TOPNAV_COLLAPSED)}>
+            {navItems}
+          </div>
+          <ReactCSSTransitionGroup
+            className={classnames(cssClass.SUBNAV, openChild && cssClass.SUBNAV_OPEN)}
+            transitionEnterTimeout={400}
+            transitionLeaveTimeout={400}
```

Comment:
corresponds with the `.timingSlowly` LESS mixin we use for setting the duration for transitions that we're doing purely in CSS (in animation.less).

would be nice to have those constants exported to JS as well, so we can reference them.
in the meantime, gonna create a constant for this so it's less confusing for people reading this. thanks for flagging - https://github.com/Clever/components/pull/185/commits/dbb905e429f7774b3aaff3204be0a448774cc4aa